versions/s-/srpc.json

```
@@ -0,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "a661dc3cfb7479653f6db0ca948bb5bda1588d63",
```

```suggestion
      "git-tree": "7f8e905bb42ba10b9076c4fca512933f4e3171e5",
```
The value has changed. You can run `vcpkg x-add-version srpc` to update.